### PR TITLE
Form b form c questions

### PR DESCRIPTION
**Pull request by eleanorreem**
- renamed demoform
- put real  questions in
- note that I haven't thought of a better name for setA setB and setC forms. I think they are all interchangable so not sure what would be better to do.

### CODE REVIEW

File: components/forms/AboutYouSetBForm.tsx [hunk 1]
```diff
@@ -69,42 +106,46 @@ const AboutYouSetBForm = () => {
...
-          logEvent(ABOUT_YOU_SETA_ERROR, {
-            ...eventUserData,
-            message: error,
-          });
-          setLoading(false);
-          throw error;
-        });
-    }
+
+    // if (process.env.NEXT_PUBLIC_ZAPIER_WEBHOOK_SETB_FORM) {
```
Comment (swetha-charles) [inline components/forms/AboutYouSetBForm.tsx:129]: I assume this will be uncommented when the zaps are set up - is that correct?

File: components/forms/AboutYouSetBForm.tsx [hunk 2]
```diff
@@ -29,11 +22,12 @@ const actionsStyle = {
 
 const AboutYouSetBForm = () => {
   const t = useTranslations('Account.aboutYou.setBForm');
+  const tSetA = useTranslations('Account.aboutYou.setAForm');
```
Comment (swetha-charles) [inline components/forms/AboutYouSetBForm.tsx:25]: Could we add a "shared" object with the "aboutYou" object? So that we're not pulling set A text into set B.

File: components/forms/AboutYouSetBForm.tsx [hunk 3]
```diff
@@ -29,11 +22,12 @@ const actionsStyle = {
...
+
   const router = useRouter();
 
   const [eventUserData, setEventUserData] = useState<any>(null);
   const [loading, setLoading] = useState<boolean>(false);
-  const [hopesInput, setHopesInput] = useState<string>('');
   const [scale1Input, setScale1Input] = useState<number>(3);
```
Comment (swetha-charles) [inline components/forms/AboutYouSetBForm.tsx:31]: Might be worth pulling out "3" as a constant e.g. `const DEFAULT_SCALE_START = 3;`

File: components/forms/AboutYouSetCForm.tsx [hunk 4]
```diff
@@ -29,6 +22,8 @@ const actionsStyle = {
 
 const AboutYouSetCForm = () => {
   const t = useTranslations('Account.aboutYou.setCForm');
+  const tSetA = useTranslations('Account.aboutYou.setAForm');
```
Comment (swetha-charles) [inline components/forms/AboutYouSetCForm.tsx:25]: Same "shared" comment

File: components/forms/AboutYouSetCForm.tsx [hunk 5]
```diff
@@ -61,50 +69,77 @@ const AboutYouSetCForm = () => {
...
+    logEvent(ABOUT_YOU_SETC_REQUEST, eventUserData);
     const formData = {
       date: new Date().toISOString(),
       user_id: user.id && hashString(user.id),
       hopes: hopesInput,
       ...eventUserData, // add user data
     };
-
     scaleQuestions.forEach((question) => {
       formData[question.name.toLowerCase()] = question.inputState;
```
Comment (swetha-charles) [inline components/forms/AboutYouSetCForm.tsx:119]: `formData` will need to have the `sinceBloomScaleQuestions` set as well I think. 

You maybe be doing this already in a separate PR in which case go ahead and resolve.

File: messages/account/es.json [hunk 6]
```diff
@@ -34,8 +34,9 @@
       "titleA": "Lo que quieres de Bloom",
       "descriptionALine1": "¡Gracias por tus respuestas!",
       "descriptionALine2": "Como eres un usuario nuevo, nos gustaría saber cómo te encuentras antes de usar nuestros servicios. Estas preguntas nos ayudan a medir el impacto que tiene Bloom sobre las personas sobrevivientes que lo usan, y nos ayuda a enfocarnos en las cosas correctas.",
+      "setBAndCDescription": "As a Bloom user, we'd like to understand how you're doing now that you’ve been using our services. Gathering this information helps us to measure the impact that using Bloom is having for survivors, and in turn helps us to focus on the right things.",
       "goToCourses": "Saltar e ir a cursos",
-      "demoForm": {
+      "demographicForm": {
```
Comment (swetha-charles) [inline messages/account/es.json:39]: 👌  much quicker to understand now lol

File: components/forms/AboutYouSetCForm.tsx [hunk 7]
```diff
@@ -29,6 +22,8 @@ const actionsStyle = {
 
 const AboutYouSetCForm = () => {
   const t = useTranslations('Account.aboutYou.setCForm');
+  const tSetA = useTranslations('Account.aboutYou.setAForm');
+
   const router = useRouter();
```
Comment (swetha-charles) [inline components/forms/AboutYouSetCForm.tsx:27]: The router variable isn't used so can be removed

File: components/forms/AboutYouSetCForm.tsx [hunk 8]
```diff
@@ -61,50 +69,77 @@ const AboutYouSetCForm = () => {
...
     event.preventDefault();
     setLoading(true);
-
-    logEvent(ABOUT_YOU_SETA_REQUEST, eventUserData);
-
+    logEvent(ABOUT_YOU_SETC_REQUEST, eventUserData);
     const formData = {
       date: new Date().toISOString(),
       user_id: user.id && hashString(user.id),
       hopes: hopesInput,
```
Comment (swetha-charles) [inline components/forms/AboutYouSetCForm.tsx:115]: Do we have a "hopes" part for set C? I'm not sure so thought I'd ask. 

If we don't have it, we can remove it from here as well as removing the state and the setter.

**Review by swetha-charles — COMMENTED** [review]
Let me know if you want to chat through!

File: components/forms/AboutYouSetCForm.tsx (hunk 8 above)
Comment (eleanorreem) [reply components/forms/AboutYouSetCForm.tsx:115]: Nope!

File: pages/account/about-you.tsx [hunk 9]
```diff
@@ -95,16 +95,19 @@ const AboutYou: NextPage = () => {
           <Typography variant="h2" component="h2">
             {t('header')}
           </Typography>
-          {questionSetParam === 'default' ? ( // TODO this might need to change if there are different descriptions for different forms
-            <Typography>{t('description')}</Typography>
-          ) : (
+
+          {questionSetParam === 'default' && <Typography>{t('description')}</Typography>}
```
[the remark this answers is not in the document]
Comment (eleanorreem) [reply pages/account/about-you.tsx:99]: Sure makes sense

File: components/forms/AboutYouSetBForm.tsx (hunk 2 above)
Comment (eleanorreem) [reply components/forms/AboutYouSetBForm.tsx:25]: Yeah thought about it but didn't do it. 😆  Will do it now though

**Review by swetha-charles — DISMISSED** [review]
